lib/dry/struct/class_interface.rb

```diff
@@ -361,6 +314,13 @@ def meta(meta = Undefined)
           end
         end
       end
+
+      # Stores an object for building nested struct classes
+      # @return [StructBuilder]
+      def struct_builder
+        @struct_builder ||= StructBuilder.new(self)#.freeze
```

Comment:
Did you mean to do something with the `#.freeze` here, @flash-gordon?

Reply:
@timriley yeah, to remove, thanks for pointing :)